utils/releaser/src/FilesProvider/DockerComposeFilesProvider.php

```diff
@@ -15,7 +15,7 @@ public function provide(): array
     {
         $finder = Finder::create()
             ->in(getcwd())
-            ->exclude('vendor')
+            ->exclude(['vendor', 'project-base/var/postgres-data', 'utils/releaser'])
             ->files()
             ->name('#docker-compose([\w-]+)?\.yml\.dist$#');
```

Comment:
just for some info, we have also `project-base/docker/conf/docker-compose.prod.yml.dist` but there is no tag or version record.

Reply:
I see, so then it is not affected by `DockerComposeFileManipulator::processFileToString()`, so everything is ok, right? :slightly_smiling_face: